templates.go

```diff
@@ -69,6 +69,12 @@ var (
 	serviceDialOptionsMtx sync.Mutex
 )
 
+func NewListeners() {
+{{range $lis := .Listeners}}
+	{{$lis}} = bufconn.Listen(100)
+{{- end}}
```

Comment:
cool, I didn't know of this syntax :) Looks like we can also use `- range` above to avoid the leading newline 😁

Reply:
Oh yeah, I'll fix it on the other place as well to keep it consistent. 🙂

Reply:
Fixed.